libgraph/include/katana/TypedPropertyGraph.h

```diff
@@ -275,6 +275,11 @@ class TypedPropertyGraphView : public PGView {
       const std::vector<std::string>& edge_properties);
   static Result<TypedPropertyGraphView<PGView, NodeProps, EdgeProps>> Make(
       const PGView& pg_view);
+  static Result<TypedPropertyGraphView<PGView, NodeProps, EdgeProps>> Make(
+      PropertyGraph* pg, const std::vector<std::string>& node_properties,
+      const std::vector<std::string>& edge_properties,
+      const RDGTopology::EdgeSortKind edge_sort_kind,
```

Comment:
I'm against this change. It muddies the interface for all analytics apps using `TypedPropertyGraphView` and adds burden on the user to provide the `sort kind` which was so far hidden. Let's find a way that does not change existing interface for analytics apps.

Reply:
I can think of two options.
1) Use default value; IMO, it is equivalent to the old version
2) Use `enable_if` or `enable_if_t` to make different execution paths and hide from users?

Do you have any better suggestion?
This was also my question and my conclusion was we need a user parameter since it is algorithmic dependent information.